src/components/CompareResults/ResultsMain.tsx

```diff
@@ -1,25 +1,46 @@
-import { Suspense, useState } from 'react';
+import { Suspense, useEffect, useState } from 'react';
 
+import { SelectChangeEvent } from '@mui/material';
 import Box from '@mui/material/Box';
 import CircularProgress from '@mui/material/CircularProgress';
 import { Container } from '@mui/system';
+import { useSearchParams } from 'react-router-dom';
 import { Await, useLoaderData } from 'react-router-dom';
 import { style } from 'typestyle';
 
 import { useAppSelector } from '../../hooks/app';
 import { Colors, Spacing } from '../../styles';
 // import type { CompareResultsItem } from '../../types/state';
+import { Repository } from '../../types/state';
+import { Framework } from '../../types/types';
 import DownloadButton from './DownloadButton';
 import type { LoaderReturnValue } from './loader';
+import FrameworkDropdown from './ResultsFrameWorkDropDown';
```

Comment:
```suggestion
import ResultsFrameworkDropdown from './ResultsFrameworkDropDown';
``` What do you think about this nit change so that it's more clear this is not the same component as the framework dropdown from the search page?